src/main/resources/org/broadinstitute/hellbender/utils/helpTemplates/generic.template.html

```diff
@@ -65,7 +65,7 @@
 	<#macro argumentDetails arg>
 		<hr style="border-bottom: dotted 1px #C0C0C0;" />
 		<h3><a name="${arg.name}">${arg.name} </a>
-			<#if arg.synonyms??> / <small>${arg.synonyms}</small></#if>
+			<#if arg.synonyms != "NA"> / <small>${arg.synonyms}</small></#if>
```

Comment:
Oh, this might not be safe without the "??" guard, which checks to see if the variable exists. If it doesn't (i.e, maybe positional args IIRC), this will throw.

Reply:
Ooh, ok, I'll add that check back as well.

Reply:
Added

Reply:
Looks good now thx.